README.md

```diff
@@ -101,43 +101,40 @@ data directory.
 `tezos-{accuser, endorser}` have configurable node address, so that they can be used with both
 remote and local node.
 
-### Systemd units on other systems
+<!-- TODO: Currently this instructions are broken -->
```

Comment:
What are we going to do with it and when?

Reply:
I'll fix it soon

Reply:
Should be fixed now